views/partials/top_nav.handlebars

```diff
@@ -55,6 +55,11 @@
           FAQ
         </a>
       </li>
+      <li class="nav-item active">
+        <a class="nav-link i18n" data-i18n="ftm-nav-partners" href="javascript:;" onclick="window.location='partners'+window.location.search;">
```

Comment:
I think this point was brought up in slack earlier for the whoweare page but this isn't great for crawlers. Can you either fix (set href with current country + locale in render-nav.js) or create a new issue for that?

Reply:
let's just create a new issue to fix all of those